modules/snmp/README.md

```diff
@@ -6,219 +6,254 @@ sidebar_label: "SNMP"
-      port: 1161
       version: 2
-      max_request_size: 60
-    community: "public"
     charts:
-      - title: "example"
-        priority: 1
+      - id: "bandwidth_port"
+        title: "Switch Bandwidth for port"
         units: "kilobits/s"
         type: "area"
-        family: "lan"
-        multiply_range: [1,5]
+        family: "ports"
+        multiply_range: [1, 24]
         dimensions:
           - name: "in"
-            oid: ".1.3.6.1.2.1.2.2.1.10"
+            oid: "1.3.6.1.2.1.2.2.1.10"
             algorithm: "incremental"
             multiplier: 8
             divisor: 1000
           - name: "out"
-            oid: ".1.3.6.1.2.1.2.2.1.16"
+            oid: "1.3.6.1.2.1.2.2.1.16"
             multiplier: -8
             divisor: 1000
 ```
 
-## SNMPv3
-To use SNMPv3:
-
--   use `user` instead of `community`
--   set `options.version` to 3
-
-Example: refer to the first example in this README.
-
-Security levels (`user.level`):
-
--   1 is `noAuthNoPriv`
--   2 is `authNoPriv`
--   3 is `authPriv`
-
-Authentication protocols (`user.auth_proto`):
-
--   "1" is `none`
--   "2" is `md5`
--   "3" is `sha`
--   "4" is `sha224`
--   "5" is `sha256`
--   "6" is `sha384`
--   "7" is `sha512`
+## Data collection speed
 
-Privacy protocols (`user.priv_proto`):
+Keep in mind that many SNMP switches and routers are very slow. They may not be able to report values per second.
+`go.d.plugin` reports the time it took for the SNMP device to respond when executed in the [debug](#troubleshooting)
+mode.
 
--   "1" is `none`
--   "2" is `des`
--   "3" is `aes`
--   "4" is `aes192`
--   "5" is `aes256`
+Also, if many SNMP clients are used on the same SNMP device at the same time, values may be skipped. This is a problem
+of the SNMP device, not this collector. In this case, consider reducing the frequency of data collection (
+increasing `update_every`).
 
-## Testing the configuration
+## Finding OIDs
 
-To test it, you can run:
+Use `snmpwalk`, like this:
 
 ```sh
-/usr/libexec/netdata/plugins.d/go.d.plugin -d 1 -m snmp
+snmpwalk -t 20 -O fn -v 2c -c public 192.0.2.1
 ```
 
-The above will run it on your console and you will be able to see what Netdata sees, but also errors.
-If it works, restart Netdata to activate the snmp collector and refresh the dashboard (if your SNMP device responds with a delay, you may need to refresh the dashboard in a few seconds).
+- `-t 20` is the timeout in seconds.
+- `-O fn` will display full OIDs in numeric format.
+- `-v 2c` is the SNMP version.
+- `-c public` is the SNMP community.
+- `192.0.2.1` is the SNMP device.
 
-## Data collection speed
+## Troubleshooting
 
-Keep in mind that many SNMP switches and routers are very slow. They may not be able to report values per second. If `go.d.plugin` is executed in `debug` mode, it will report the time it took for the SNMP device to respond.
+To troubleshoot issues with the `snmp` collector, run the `go.d.plugin` with the debug option enabled. The output should
```

Comment:
We'll leave this section as is for now. We do have a component planned for debugging, so let's see how well it works for this one. :)